doc/server/api_game/api.yaml

```diff
@@ -609,6 +622,21 @@ components:
             $ref: '#/components/schemas/PerformedActionDTO'
       required: [ id, name, location, media_references ]
```

Comment:
classification can be added to required properties

Reply:
Done!